src/utils/download-file.ts

```diff
@@ -73,7 +73,7 @@ export const loadDocumentList = async (filing: ApiResponseFilingI) => {
           }
         } else if (groupName === 'uploadedCourtOrder') {
           const fileNumber = filing.data?.order?.fileNumber || unknownStr
-          const title = fileNumber === unknownStr ? `${filing.displayName} ${fileNumber}` : `${filing.displayName}`
+          const title = isCourtOrderType(filing) ? `${filing.displayName} ${fileNumber}` : `${filing.displayName}`
```

Comment:
This was a line of code that you changed in #142.

![image](https://github.com/user-attachments/assets/4b8e72e0-0c20-4920-ae2c-e830f5c53337)

I compared these changes with this change, and I think it's good 👍 This makes sense!